gamemode/core/derma/cl_information.lua

```diff
@@ -151,7 +156,9 @@ function PANEL:Init()
 				self.description:SetMouseInputEnabled(true)
 				self.description:SetCursor("hand")
 				self.description.DoClick = function(this)
-					RunConsoleCommand("ix", "CharDesc")
+					Derma_StringRequest(lDesc, lCmdCharDesc, self.description:GetText(), function(desc)
```

Comment:
Won't make any performance difference whatsoever if you just put L"phrase_name" directly here as an anonymous variable.

Reply:
I had issues with line length. I will just break it into multiple lines

Reply:
It'll actually be more efficient in the function anyway because it's not needed until the string request.